src/bot/gh/mod.rs

```diff
@@ -255,7 +255,7 @@ impl CodePermalink {
 
         Ok(code
             .lines()
-            .skip(l1)
+            .skip(l1 - 1)
```

Comment:
アンダーフロー / オーバーフロー対策
```suggestion
            .skip(l1.saturating_sub(1))
```

253 行で同じミスしちゃってるのでついでに治しておいてもらえると幸いです (加算は `saturating_add`)